core/comments/comment_view.ts

```diff
@@ -124,6 +131,14 @@ export class CommentView implements IRenderedElement {
     this.moveTo(new Coordinate(0, 0));
   }
 
+  private createHighlightRect(svgRoot: SVGGElement): SVGRectElement {
```

Comment:
Missing tsdoc

Reply:
Do we want to turn the lint warnings for this back on?

Reply:
Nope.